lib/src/mt_ptp.c

```diff
@@ -739,8 +820,12 @@ static int ptp_parse_follow_up(struct mt_ptp_impl* ptp,
         ptp->t2_sequence_id);
     return -EINVAL;
   }
-
+#ifndef WINDOWSENV
```

Comment:
Why need a WINDOWSENV condition here? No `msg->hdr.correction_field` for windows?

Reply:
> Why need a WINDOWSENV condition here? No `msg->hdr.correction_field` for windows?

Because Windows lacks be64toh() function.

Reply:
Not suggest write such base algorithm again. Especially write code by AI, I am not sure it run as expectation or not.

The be64toh() function should be ready in POSIX like system. The function is defined in endian.h. However, MSYS2 being used by out CI lacks the endian.h. Is it possible to ask CI administrator to check this issue?

Reply:
Not suggest write such base algorithm again. Especially write code by AI, I am not sure it run as expectation or not.

The be64toh() function should be ready in POSIX like system. The function is defined in endian.h. However, MSYS2 being used by out CI lacks the endian.h. Is it possible to ask CI administrator to check this issue?

Reply:
Done by adding `be64toh` definition from https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-paho.mqtt.c/002-fix-build.patch. The code is BSD License.